### DISCUSSION TITLE
Using NocoDB : Migrating Airtable to SQL database with attachments [Jun 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2348] [Discord Threads]
> 🙋‍♂️ **cirsten**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Which version are you using ? It should not take so much of time.  
#  
> 💁 **o1lab**  
#### I will be happy to help & DM you  
#  
> 🙋‍♂️ **cirsten**  
#### Using version 0.91.7_7.0.16. about to head out , sorry  
#  
> 💁 **o1lab**  
#### was able to migrate  
![image0](https://cdn.discordapp.com/attachments/982924659651641354/982936822101454918/Screenshot_2022-06-05_at_10.19.36.png)  
#  
> 💁 **o1lab**  
#### please send your project info (right top menu)  
#  
> 💁 **o1lab**  
#### and docker logs  
#  
> 💁 **o1lab**  
#### I wonder if has to do with truenas  
#  
> 🙋‍♂️ **cirsten**  
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **pg**   
#### ProjectOnRootDB: **false**   
#### RootDB: **pg**   
#### PackageVersion: **0.91.7**  
#  
> 🙋‍♂️ **cirsten**  
#### I three logs  
#  
> 🙋‍♂️ **cirsten**  
#### Not sure if my logs are uploading?  
#  
> 💁 **o1lab**  
#### nope  
#  
> 💁 **o1lab**  
#### they are not  
#  
> 💁 **o1lab**  
#### you can send me to my email  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/982924659651641354)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
